integrations/applications/google-workspace/resources/info.json

```diff
@@ -0,0 +1,20 @@
+{
+	"id": "google-workspace",
+	"version": "1.0.8-SNAPSHOT",
```

Comment:
This version should be 1.0.0, because this is independent from the component versions. Please check other templates and change the versions.